src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -486,13 +486,13 @@ public function tempnam($dir, $prefix)
     {
         list($scheme, $hierarchy) = $this->getSchemeAndHierarchy($dir);
 
-        // If no scheme or scheme is "file" create temp file in local filesystem
-        if (null === $scheme || 'file' === $scheme) {
+        // If no scheme or scheme is "file" or "gs" create temp file in local filesystem
```

Comment:
If others agree, we could add a brief help message explaining what `gs` is. Instead of this:

```
// If no scheme or scheme is "file" or "gs" create temp file in local filesystem
```

Something like this:

```
// If no scheme or scheme is "file" or "gs" (Google Cloud) create temp file in local filesystem
```